tests/README.md

```diff
@@ -61,7 +61,7 @@ this decorator here](https://github.com/ethereum/consensus-specs/blob/dev/tests/
 @spec_state_test
 ```
 
-[This decorator](https://github.com/qbzzt/consensus-specs/blob/dev/tests/core/pyspec/eth2spec/test/context.py#L232-L234) specifies
+[This decorator](https://github.com/ethereum/consensus-specs/blob/dev/tests/core/pyspec/eth2spec/test/context.py#L232-L234) specifies
```

Comment:
Now the link points to 

```
def large_validator_set(spec: Spec):
    """
    Helper method to create a large series of default balances.
```

Which does not appear to relate to the docs

Reply:
I think it should be:

* https://github.com/ethereum/consensus-specs/blob/6c581ca42c7e98fbdee7beed5643a41b874100d2/tests/core/pyspec/eth2spec/test/context.py#L291-L293

But I would actually be in favor of removing the link. Doesn't seem that useful to me.